queryresultio/text/src/main/java/org/eclipse/rdf4j/query/resultio/text/tsv/SPARQLResultsTSVReader.java

```diff
@@ -0,0 +1,22 @@
+package org.eclipse.rdf4j.query.resultio.text.tsv;
```

Comment:
Editorial: class is missing a license header

Reply:
Added license header.